doc/internal/man3/evp_keymgmt_util_export_to_provider.pod

```diff
@@ -48,6 +59,11 @@ evp_keymgmt_export_to_provider() and evp_keymgmt_util_fromdata()
 return a pointer to the appropriate provider side key (created or
 found again), or NULL on error.
 
+evp_keymgmt_util_find_operation_cache_index() returns the index of the
+operation cache slot.  If there was no matching slot, it returns the
+index of the first matching slot, as if I<keymgmt> is NULL, or the
```

Comment:
first matching slot is not right - maybe just use the text I had above (or something similar)